src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
that's the part where I wasn't sure. I almost feel that we shouldn't be forcing either uppercase or lowercase, and instead use whatever was specified by the programmer. If someone created the column names with quotes (outside of the migration) using lowercase, it will fail if we uppercase it.